hoomd/md/test/test_harmonic_dihedral_force.cc

```diff
@@ -52,7 +52,7 @@ void dihedral_force_basic_tests(dihedralforce_creator tf_creator, std::shared_pt
 
```

Comment:
You should add a unit test either here (in the C++ code) or at the python level where you check that the forces are computed correctly if `phi_0` is not zero. Right now, I don't think there are any checks for this, so we can't be sure that the implementation is right.

Reply:
Okay see `dihedral_force_phase_shift`, sort of a contrived but simple example, I think